Jint/Runtime/Interop/ObjectWrapper.cs

```diff
@@ -364,6 +364,10 @@ private PropertyDescriptor GetOwnProperty(JsValue property, bool mustBeReadable,
         }
 
         var accessor = _engine.Options.Interop.TypeResolver.GetAccessor(_engine, ClrType, member, mustBeReadable, mustBeWritable);
+        if (accessor == ConstantValueAccessor.NullAccessor)
```

Comment:
Should this check contain also a check that ClrType != Target.GetType() to clarify intent?

Reply:
Done